docs/sources/source/nlp/dialogue_state_tracking.rst

```diff
@@ -427,78 +427,81 @@ Note, during the evaluation, the model first generates predictions and writes th
 Model Improvements
 ------------------
 
-Model improvements added to get better performance results and increase model flexibility:
+Model features added to get better performance results or increase model flexibility:
 
 - data augmentation
 - system retrieval mechanism
 - ability to make schema embeddings trainable during the model training
+- adding an attention layer between the encoder and the model heads
+
+Data augmentation is done offline with `examples/nlp/dialogue_state_tracking/data/sgd/dialogue_augmentation.py <https://github.com/NVIDIA/NeMo/blob/master/examples/nlp/dialogue_state_tracking/data/sgd/dialogue_augmentation.py>`_.
+We used 10x as augmentation factor.
+
+The following table shows results of the SGD baseline and that of some NeMo model features. The focus was to improve seen services.
+We use `*` to denote the error fixed in NeMo that occurred in the original TensorFlow implementation of SGD for single domain.
```

Comment:
maybe call it an `issue` not an `error` and add a description of what was wrong and how it affected the baseline?